code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -258,7 +258,7 @@
 		return
 	var/list/modules = list()
 	modules.Add(robot_module_types)
-	if((crisis && security_level == SEC_LEVEL_RED) || crisis_override) //Leaving this in until it's balanced appropriately.
+	if(security_level == SEC_LEVEL_RED || SEC_LEVEL_DELTA) //There are no brakes in the combat module train.

```

Comment:
Erm, I don't think this will work as intended.

It reads: **IF** security_level == SEC_LEVEL_RED **OR IF** SEC_LEVEL_DELTA (which is always true).
I think the proper syntax is: `if (security_level == (SEC_LEVEL_RED | SEC_LEVEL_DELTA))`? I forget how to mark such a thing up in DM.